core/lib/storage/src/db.rs

```diff
@@ -304,7 +307,8 @@ impl<CF: NamedColumnFamily> RocksDB<CF> {
 
     pub fn with_options(path: &Path, options: RocksDBOptions) -> Result<Self, rocksdb::Error> {
         let caches = RocksDBCaches::new(options.block_cache_capacity);
-        let db_options = Self::rocksdb_options(None, None);
+        let mut db_options = Self::rocksdb_options(None, None);
+        db_options.set_max_open_files(options.max_open_files);
```

Comment:
Just to be sure: is this setting enforced on the database level, rather than on the column family level (options for CFs are created below)?

Reply:
Yes - actually I'd prefer it on process level (in case the process has multiple RocksDB instances), but that's probably too complex...